package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "css-frameworks",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "sass src/assets/scss:dist/css",
+    "watch": "sass --watch src/assets/scss:dist/css & live-server"
```

```suggestion
    "watch": "sass --watch src/assets/scss:dist/css"
```

Since you don't have this installed you should remove it from this script.

---

It was not added yet, but is now (at least for index.html) :) I use live-server, but as an vscode extension.